tests/Makefile.test

```diff
@@ -0,0 +1,23 @@
+### Build commands
+### ! Please note that you need to change the port and path of Unity as appropriate !
+
+### Unit tests
+
+##CAN
+
+# 1 board, no wire
+make FQBN=Infineon:xmc:XMC1400_XMC2GO PORT=COM42 UNITY_PATH=\Unity test_can_single monitor
```

Comment:
If this is part of the arduino-core-test, should it be (documented) there?

Reply:
It's just a make command for developer reference. Removed considering that this is strongly project related.
Can also move back